lib/SauceBrowser.js

```diff
@@ -199,7 +199,7 @@ SauceBrowser.prototype.shutdown = function(err) {
 
     self.stopped = true;
 
-    finish_shutdown = function() {
+    self.finish_shutdown = function() {
```

Comment:
Hmm. I would've probably just made a var out of this. Not sure it feels right to put a function to `self` within a function.